addons/mail/static/tests/thread/thread.test.js

```diff
@@ -315,13 +313,9 @@ test("mark channel as fetched when a new message is loaded and as seen when focu
     );
     await contains(".o-mail-Message");
     await assertSteps(["rpc:channel_fetch"]);
-    await contains(".o-mail-Thread-newMessage hr + span", { text: "New messages" });
-    await focus(".o-mail-Composer-input");
-    await contains(".o-mail-Thread-newMessage hr + span", { count: 0, text: "New messages" });
-    await assertSteps(["rpc:set_last_seen_message"]);
 });
 
-test("mark channel as fetched and seen when a new message is loaded if composer is focused [REQUIRE FOCUS]", async () => {
+test("mark channel as fetched when a new message is loaded if thread is focused [REQUIRE FOCUS]", async () => {
```

Comment:
Test was already not super-well worded. Considering you make some diff in the naming, you could replace the `if` by `and`. It saves a few seconds each time I have to read this test name :)